recipes/apr-util/all/conanfile.py

```diff
@@ -0,0 +1,204 @@
+            "--with-mysql={}".format(my_unix_path(self.deps_cpp_info["libmysqlclient"].rootpath)) if self.options.with_mysql else "--without-mysql",
+            "--with-pgsql={}".format(my_unix_path(self.deps_cpp_info["libpq"].rootpath)) if self.options.with_postgresql else "--without-pgsql",
+            "--with-sqlite3={}".format(my_unix_path(self.deps_cpp_info["sqlite3"].rootpath)) if self.options.with_sqlite3 else "--without-sqlite3",
+            "--with-ldap={}".format(my_unix_path(self.deps_cpp_info["ldap"].rootpath)) if self.options.with_ldap else "--without-ldap",
+            "--with-berkeley-db={}".format(my_unix_path(self.deps_cpp_info["libdb"].rootpath)) if self.options.dbm == "db" else "--without-berkeley-db",
+            "--with-gdbm={}".format(my_unix_path(self.deps_cpp_info["gdbm"].rootpath)) if self.options.dbm == "gdbm" else "--without-gdbm",
+            "--with-ndbm={}".format(my_unix_path(self.deps_cpp_info["ndbm"].rootpath)) if self.options.dbm == "ndbm" else "--without-ndbm",
+
+        ]
+        if self.options.dbm:
+            conf_args.append("--with-dbm={}".format(self.options.dbm))
+        self._autotools.configure(args=conf_args, configure_dir=self._source_subfolder)
+        return self._autotools
+
+    def _patch_sources(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
+    def build(self):
+        if self.options.shared != self.options["apr"].shared:
+            raise ConanInvalidConfiguration("apr-util must be built with same shared option as apr")
+
+        self._patch_sources()
+        if self.settings.os == "Windows":
+            cmake = self._configure_cmake()
+            cmake.build()
+        else:
+            autotools = self._configure_autotools()
+            autotools.make()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        if self.settings.os == "Windows":
+            cmake = self._configure_cmake()
+            cmake.install()
+        else:
+            autotools = self._configure_autotools()
+            autotools.install()
+
+            for file in glob.glob(os.path.join(self.package_folder, "lib", "apr-util-1", "*.la")):
+                os.unlink(file)
+            os.unlink(os.path.join(self.package_folder, "lib", "libaprutil-1.la"))
+            tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info(self):
+        self.cpp_info.names["pkg_config"] = "apr-util-1"
+        self.cpp_info.libs = ["libaprutil-1" if self.settings.os == "Windows" and self.options.shared else "aprutil-1"]
+        self.cpp_info.libdirs.append(os.path.join("lib", "apr-util-1"))
+        if not self.options.shared:
+            self.cpp_info.defines = ["APU_DECLARE_STATIC"]
+            if self.settings.os == "Linux":
+                self.cpp_info.system_libs = ["dl", "pthread", "uuid", "rt"]
```

Comment:
I think apr-util can build fine without uuid, so this uuid should be either magically conditional or the recipe itself should depend on uuid.

Reply:
Your current recipe also fails on my ubuntu machine.
``` 
  cannot find -luuuid
```
so uuuid is definitely not a piece of standard package.